.gitignore

```diff
@@ -171,3 +171,5 @@ packages/service-integration/compose-spec.json
 
 # temporary files by vscode
 Untitled*
+
+_tmp/
```

Comment:
not a good idea. Check comments about tempdir below

Reply:
NOTE2: when you want to create an artifact and want to ignore it, just add in the name the keyword ``ignore`` 
for example ``myfile.txt`` -> ``myfile.ignore.txt``